doc/ArchInstall/arch-chroot_install.sh

```diff
@@ -162,22 +162,16 @@ systemctl enable wpa_supplicant@${WIFI_INTERFACE}
 # ZSH SHELL #
 #############
 
-# Change user shell to zsh
+# Change user shell to zsh and create a blank config file to prevent user prompt
 chsh -s /usr/bin/zsh ${USER}
+touch ${HOME}/.zshrc
 
-############
-# SSH KEYS #
-############
-
-# Create user ssh directory
-mkdir -p ${HOME}/.ssh
-
-# Register docker client as an authorized user
-wget https://raw.githubusercontent.com/NUbots/NUbots/main/docker/home/nubots/.ssh/id_rsa.pub \
-    -O ${HOME}/.ssh/authorized_keys
```

Comment:
I thought we needed this, how do we ssh into the robot without this?

Reply:
Using the password I assume.
Also the new docker image just forwards the ssh credentials from the host